containers/Dockerfile.EESSI-client-pilot-centos7

```diff
@@ -1,35 +1,28 @@
 ARG cvmfsversion=2.8.2
```

Comment:
Same here, let's also bump this to `2.9.1`

Reply:
Done!